websub-ballerina/src/websub_generic_integration_tests/tests/integration-tests/subscriber/test_custom_subscribers.bal

```diff
@@ -0,0 +1,395 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// -- Done
```

Comment:
```suggestion
```

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-websub/pull/1/commits/7defc19195f199d3899556e82f9a35eb1ebf4bd5